File: frontend/src/app/dto/product-customer/product-customer.insert.list.req.dto.ts
import { ProductCustomerInsertReqDto } from "./product-customer.insert.req.dto";

export class ProductCustomerInsertListReqDto{

    private data! :ProductCustomerInsertReqDto[];

    public get _data(): ProductCustomerInsertReqDto[] {
        return this.data;
    }

    public set _data(_data: ProductCustomerInsertReqDto[]) {
        this.data = _data;
    }


    

}

File: backend/src/main/java/com/lawencon/ticketms/dto/productcustomer/ProductCustomerInsertListReqDto.java
package com.lawencon.ticketms.dto.productcustomer;

import java.util.List;

public class ProductCustomerInsertListReqDto {
	
	private List<ProductCustomerInsertReqDto> data;

	public List<ProductCustomerInsertReqDto> getData() {
		return data;
	}

	public void setData(List<ProductCustomerInsertReqDto> data) {
		this.data = data;
	}
	
}
